### DISCUSSION TITLE
汉化后代码无法推送该怎么解决？

### DISCUSSION THREAD

**Discussion by myorange2009** [lkyero/GitHubDesktop_zh#57] [General]
汉化后，代码无法推送
报以下错误：
error: cannot spawn C:\Users\Administrator\AppData\Local\GitHubDesktop\app-2.9.6\resources\app\static\ask-pass-trampoline.bat: No such file or directory
fatal: could not read Username for 'https://github.com': terminal prompts disabled

**Comment by lonelyleaves** [comment]
很好的项目 可惜没维护了，如果能做成poedit模式汉化就好了
